precise/iaf_psc_alpha_ps.h

```diff
@@ -0,0 +1,548 @@
+spikes is determined by interpolation once a threshold crossing has
+been detected. Return from refractoriness occurs precisly at spike
+time plus refractory period.
+
+This implementation is more complex than the plain iaf_psc_alpha
+neuron, but achieves much higher precision. In particular, it does not
+suffer any binning of spike times to grid points. Depending on your
+application, the canonical application may provide superior overall
+performance given an accuracy goal; see [1] for details.  Subthreshold
+dynamics are integrated using exact integration between events [2].
+
+
+Parameters:
+
+The following parameters can be set in the status dictionary.
+
+V_m          double - Membrane potential in mV
+E_L          double - Resting membrane potential in mV.
+V_min        double - Absolute lower value for the membrane potential.
+C_m          double - Capacity of the membrane in pF
+tau_m        double - Membrane time constant in ms.
+t_ref        double - Duration of refractory period in ms.
+V_th         double - Spike threshold in mV.
+V_reset      double - Reset potential of the membrane in mV.
+tau_syn_ex   double - Rise time of the excitatory synaptic function in ms.
+tau_syn-in   double - Rise time of the inhibitory synaptic function in ms.
+I_e          double - Constant external input current in pA.
+Interpol_Order  int - Interpolation order for spike time:
+                      0-none, 1-linear, 2-quadratic, 3-cubic
+
+Remarks:
+
+This model transmits precise spike times to target nodes (on-grid spike
+time and offset). If this node is connected to a spike_detector, the
+property "precise_times" of the spike_detector has to be set to true in
+order to record the offsets in addition to the on-grid spike times.
+
+The iaf_psc_alpha_ps neuron accepts connections transmitting
+CurrentEvents. These events transmit stepwise-constant currents which
+can only change at on-grid times.
+
+If tau_m is very close to tau_syn_ex/in, the model will numerically behave as
+if tau_m is equal to tau_syn_ex/in, to avoid numerical instabilities.
+For details, please see doc/model_details/IAF_neurons_singularity.ipynb.
+
+
+References:
+
+[1] Morrison A, Straube S, Plesser H E, & Diesmann M (2006) Exact Subthreshold
+    Integration with Continuous Spike Times in Discrete Time Neural Network
+    Simulations. To appear in Neural Computation.
+[2] Rotter S & Diesmann M (1999) Exact simulation of time-invariant linear
+    systems with applications to neuronal modeling. Biologial Cybernetics
+    81:381-402.
+[3] Hanuschkin A, Kunkel S, Helias M, Morrison A & Diesmann M (2010)
+    A general and efficient method for incorporating exact spike times in
+    globally time-driven simulations Front Neuroinformatics, 4:113
+
+Author: Tanguy Fardet (from Diesmann, Eppler, Morrison, Plesser, Straube)
+
+Sends: SpikeEvent
+
+Receives: SpikeEvent, CurrentEvent, DataLoggingRequest
+
+SeeAlso: iaf_psc_alpha, iaf_psc_alpha_presc, iaf_psc_exp_ps
```

Comment:
Why no reference to iaf_psc_alpha_canon?

Reply:
Should I add it even though we're deprecating it?

Reply:
Maybe not.